.github/workflows/caa_build_and_push.yaml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_call:
     inputs:
       registry:
-        default: 'quay.io/confidential-containers'
+        default: 'quay.io/liudalibj'
```

Comment:
Should the registries in these workflows files be moved back to `quay.io/confidential-containers` from `quay.io/liudalibj`?

Reply:
Ah, I've spotted these are reverted in a later commit. Maybe fixup 118f8f83decc494ed0276fb702a8addbae024ba5 into this one for better commit clarity?

Reply:
yeah fixuped the commit with others.